### DISCUSSION TITLE
Red crossed rectangle when outside home directory

### DISCUSSION THREAD

**Discussion by J-Kappes** [starship/starship#5268] [Q&A] [answered]
I'm using the [Nerd Font Symbols preset](https://starship.rs/presets/nerd-font.html) and my prompt always ends on a red rectangle with a diagonal from the bottom left to the top right when I'm outside my home directory. Is that what it's supposed to look like?

I pass the [FAQ font test](https://starship.rs/faq/#why-don-t-i-see-a-glyph-symbol-in-my-prompt).

**Comment by chipbuster** [comment]
Can you post a screenshot or the output of `starship explain` when that symbol shows up?

**Comment by mickimnet** [comment]
What version of _Starship_ are you using?

**Comment by J-Kappes** [comment]
I opened this topic at work. Being at home now, I'm pretty confident that it was a font issue, since instead of that red box I have a red lock symbol:
![Screenshot from 2023-06-27 11-18-48](https://github.com/starship/starship/assets/87168492/11b82d54-562a-46ac-91c8-ebd80a9e4cd7)
I'm also sure that this is the exact same `starship explain` output I got at work when having this issue.

At home I'm on Starship 1.15.0 using JetBrainsMono NF semi-bold. I'll try to report my work setup tomorrow.

**Reply by mickimnet** [reply]
I would guess that you are using a version <1.15.0 on your work machine, as the fix for the _Nerd Fonts_, was released with 1.15.0.

**Reply by J-Kappes** [reply, marked as answer]
I'm at Starship 1.15.0 at work too. It really was my font: a year old version of JetBrainsMono NF. After redownloading it, the box became a lock. But here's a screenshot of the box for the record.
![Screenshot from 2023-06-30 08-09-31](https://github.com/starship/starship/assets/87168492/c8628c37-09e4-4702-9772-d79c83e896be)
